lib/dm-ui.js

```diff
@@ -99,6 +104,12 @@ const dmUI = (parent, profile, storage, network) => {
 
     // TODO: handle control-a, -x, etc
 
+    // TODO: might be good to have some type of mixinable behavior or default behavior on 
+    // input that can be overriden. Only reason I think it might be worth exploring
+    // is the TAB, BACKSPACE, UP, DOWN, and default cases are shared here and in main-ui.js.
+
+    // However, we want to make sure we do it gracefully because this might be overengineering, and if we ulimately decide
+    // against it I totally understand. If we go down that route, we probs want to make it as simple as possible to understand
```

Comment:
Indeed. And also I want to make the UI have a `tab to focus` that rotates the UI like in a traditional ncurses app